src/main/java/com/akademia/educators_back/controller/ProblemController.java

```diff
@@ -38,14 +51,21 @@ public ProblemTo findProblemById(@PathVariable Long id){
         return problemTo;
     }
 
-
+    /**
+     * Controller to create a new problem
```

Comment:
Ogólnie dobrze ale zawsze można jeszcze dopisać coś o tym co np taki kontroller przyjmuje/wymaga w paramterze żeby utworzyć nowy obiekt np "Controller handling request to create new Problem object. Required transfer object of NewProblemTo with no id "
Taka dokumentacja jakbyś jako ktoś kto chciał tylko na podstawie niej domyśleć się co ta metoda robi i jak z niej skorzystać żeby uzyskać zamierzony efekt

P.S. można też dać info jaki status jest zwracany kiedy się operacja powiedzie lub nie